### PR TITLE
Feat/product tab

### PR DESCRIPTION
**Pull request by JM066**

### CODE REVIEW

File: src/components/LanguageSelector/index.tsx [hunk 1]
```diff
@@ -18,15 +18,33 @@ const LOCALES: Locales[] = [
     value: "ko",
   },
 ];
-
-function LanguageSelector() {
+interface Prop {
```
Comment (BruceJi7) [inline src/components/LanguageSelector/index.tsx:21]: can this be 'Props' instead lol

File: src/components/MainSlide/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+interface Props {
+  caption?: JSX.Element;
+  title: JSX.Element;
+  description?: JSX.Element;
+  image?: JSX.Element;
+  btnMsg?: JSX.Element;
+}
+function MainSlide({ title, description, image, btnMsg }: Props) {
+  const isMobile = useDetactMobile();
```
Comment (BruceJi7) [inline src/components/MainSlide/index.tsx:18]: can this be detect instead of detact pls?

**Review by BruceJi7 — APPROVED** [review]
Approve, pending a couple of thingies

File: src/components/MainSlide/index.tsx (hunk 2 above)
Comment (JM066) [reply src/components/MainSlide/index.tsx:18]: Oh no.. again.. I should stop making typos!

**Pull request merged**